pkg/scheduler/framework/plugins/serviceaffinity/service_affinity.go

```diff
@@ -46,22 +52,31 @@ func New(plArgs *runtime.Unknown, handle framework.FrameworkHandle) (framework.P
 	nodeInfoLister := handle.SnapshotSharedLister().NodeInfos()
 	podLister := handle.SnapshotSharedLister().Pods()
 	serviceLister := informerFactory.Core().V1().Services().Lister()
-	fitPredicate, predicateMetadataProducer := predicates.NewServiceAffinityPredicate(nodeInfoLister, podLister, serviceLister, args.Labels)
 
+	fitPredicate, predicateMetadataProducer := predicates.NewServiceAffinityPredicate(nodeInfoLister, podLister, serviceLister, args.Labels)
 	// Once we generate the predicate we should also Register the Precomputation
 	predicates.RegisterPredicateMetadataProducer(predicates.CheckServiceAffinityPred, predicateMetadataProducer)
 
+	priorityMapFunction, priorityReduceFunction := priorities.NewServiceAntiAffinityPriority(podLister, serviceLister, args.Labels)
```

Comment:
should be ```args.AntiAffinityLabelsPreference```, how did the tests pass?

Reply:
I am still checking the code. Almost every line was a conflict with the other PR. I will let you know once I am confident.

Reply:
It's ready for review now.